### PR TITLE
:wrench:adjust license rule for GA and OR

### PR DESCRIPTION
**Pull request by ruchenshanghai**
asana link: https://app.asana.com/0/1134603122114360/1203131802641191/f
https://app.asana.com/0/1134603122114360/1203131687997182/f

### CODE REVIEW

File: regex.json [hunk 1]
```diff
@@ -65,9 +65,9 @@
     ]
   },
   "GA": {
-    "rule": "^[0-9]{7,9}$",
+    "rule": "^[0-9]{9}$",
     "description": [
-      "7-9 Numeric"
+      "9 Numeric"
```
Comment (sirkay777) [inline regex.json:70]: could you point to me why this decision is made? I don't find any info in the task.

Comment (ruchenshanghai) [reply regex.json:70]: image
GA needs 9 digits

Comment (sirkay777) [reply regex.json:70]: sorry, I thought it is changed from 9 to 7-9, it's actually change from 7-9 to 9.
make sense!

**Review by sirkay777 — APPROVED** [review]
lgtm

**Pull request merged**